opta/commands/deploy.py

```diff
@@ -4,7 +4,7 @@
 
 from opta.amplitude import amplitude_client
 from opta.commands.apply import _apply
-from opta.commands.push import _push, get_push_tag, is_service_config
+from opta.commands.push import _push, is_service_config
```

Comment:
I know it was already there, but functions that are to be exported should not start with an underscore-- pls change its name to push

Reply:
There already exists a `push` method present.